census/static/jquery/jquery-1.7.1.min.js

```diff
@@ -0,0 +1,4 @@
+/*! jQuery v1.7.1 jquery.com | jquery.org/license */
```

Comment:
Hmm, this is a very old version of jQuery. The latest is 3.4.1.

Can we at least try upgrading to 1.12.4? I'm sure there are security issues with an old version like this.

Reply:
My bad, I overlooked that while I was trying to get the calendar to work. I've updated it to 3.4.1 now.